.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 
 /utils/build/docker/cpp/ @DataDog/dd-trace-cpp @DataDog/system-tests-core
 /utils/build/docker/dotnet*/ @DataDog/apm-dotnet @DataDog/asm-dotnet @DataDog/system-tests-core
-/utils/build/docker/golang*/ @DataDog/apm-go @DataDog/system-tests-core
+/utils/build/docker/golang*/ @DataDog/guild-dd-trace-go @DataDog/system-tests-core
```

Comment:
Is it the good name ? https://github.com/orgs/DataDog/teams/dd-trace-go-guild

Reply:
Oops, thought I fixed it, thanks for the heads up